src/VssCommandProcessor.cpp

```diff
@@ -407,5 +407,39 @@ string VssCommandProcessor::processQuery(const string &req_json,
   return response;
 }
 
+string VssCommandProcessor::processQuery(const string &req_json) {
+  jsoncons::json root;
+  string response;
+  try 
+  {
+    root = jsoncons::json::parse(req_json);
+    string action = root["action"].as<string>();
+    logger->Log(LogLevel::VERBOSE, "Receive action: " + action);
+
+    if (action == "getMetaData") 
```

Comment:
for the first version, it is ok. But the idea to use grpc is to define remote function interface, instead of parsing string